package.json

```diff
@@ -221,8 +221,6 @@
     "fast-levenshtein": "^2.0.6",
     "file-loader": "^1.1.11",
     "local-storage-fallback": "^4.1.1",
-    "modernizr": "^3.6.0",
-    "modernizr-loader": "^1.0.1",
```

Comment:
Farewell Modernizr 👋

Reply:
This should actually fix the [babel 7 branch](https://github.com/department-of-veterans-affairs/vets-website/pull/8291) I have, I think. :crossed_fingers: